api-spec/extensions/sort/README.md

```diff
@@ -15,4 +37,21 @@ The STAC search endpoint, `/stac/search`, by default returns results in descendi
 }
 ```
 
-where <direction> is either "asc" (ascending) or "desc" (descending). The `sort` value is an array, so multiple sort fields can be defined which will be used to sort the data in the order provided (e.g., first by `datetime`, then by `eo:cloud_cover`).
+```json
+{
+    "sort": [
```

Comment:
Borrowing from OGC approaches, what about:

```json
    "sort": [
        {
            "property": "created",
            "order": "asc"
        }
```

Reply:
@tomkralidis can you point me to the documentation for those OGC approaches?

Reply:
@philvarner 

- from CSW 2.0.2: http://portal.opengeospatial.org/files/?artifact_id=20555 (Table 65)
- from CSW 3.0: http://docs.opengeospatial.org/is/12-176r7/12-176r7.html (Table 22)